source/basic.tex

```diff
@@ -5061,7 +5062,7 @@
 on the access to these entities\iref{class.access};
 
 \item
-\defnx{unions}{\idxcode{union}}, which are classes capable of containing objects of
+\textit{unions}, which are classes capable of containing objects of
```

Comment:
Why a bare `\textit` in the main document?

Reply:
Because everything else in this itemized list is italized, and other bullets claim to contain definitions, but this one (and likely the other bullets, too) are not actually definitions. I wanted to keep the local italics consistency. If you think otherwise, I'll just remove the italics here.

Reply:
Maybe use `\emph` then?

Reply:
Fixed.